### ISSUE TITLE
Cannot launch games with Proton

### ISSUE THREAD

**Issue by fubar-coder** [Heroic-Games-Launcher/HeroicGamesLauncher#794]
### Try to get help on our Wiki first: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/Troubleshooting

**Describe the bug**

I cannot launch games when I (for example) select "Proton 6.3". It seems that the launcher is unable to determine the wine version.

**Add Logs**

```text
INFO: 
 Launch Command:          __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia    /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Weaver   --wine  --wine-prefix '/home/**redacted**/.wine'  
ERROR: Error - Command failed:          __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia    /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Weaver   --wine  --wine-prefix '/home/**redacted**/.wine'  
usage: legendary launch <App Name> [options]
legendary launch: error: argument --wine: expected one argument
```

**To Reproduce**

1. Install a game
2. Change the Wine runtime to "Proton 6.3"
3. Try to start the game
4. The error above is shown in the terminal

**Expected behavior**

The game should start.

**Screenshots**

**Desktop (please complete the following information):**

- OS: Ubuntu 21.10 (amd64)
- Heroic Version 1.11.0-alpha Caesar Clown
- Game: Rebel Galaxy
- Wine/Proton version: Proton 6.3
- Hardware specs (RAM, CPU, GPU): 32GB RAM, Ryzen 2700X, NVidia GeForce RTX 2060

**Additional context**

**Comment by flavioislima** [comment]
What happens if you try using wine?
Are you selecting proton directly from the game settings?
Also try heroic beta.

**Comment by fubar-coder** [comment]
> What happens if you try using wine?

Using wine (either the default one or from lutris) works as expected.

> Are you selecting proton directly from the game settings?

This is what I did:
- Selected Proton from the app settings
- Installed the game
- Switched back to wine in the app settings
  - Game was still not startable
- Opened the game settings
  - I saw that the "Proton" selection was copied to the games settings
- Changed the game settings to "wine"
- Game was now startable

Selecting "Proton" in the game settings just makes the game unstartable again.

> Also try heroic beta.

I just tried it and it does seem to fix the problem, because the legendary launcher is now called with the "--no-wine" argument.

Thank you for your help. I'll close this ticket.
